maintainers/scripts/luarocks-packages.csv

```diff
@@ -34,6 +34,7 @@ lpty,,,,,,
 lrexlib-gnu,,,,,,
 lrexlib-pcre,,,,,,vyp
 lrexlib-posix,,,,,,
+lsqlite3complete,,,,,,
```

Comment:
could you add yourself as maintainer here ? (last column) . Rerunning the update script will update the meta.maintainers key of lsqlite3complete but it's probably faster to update it manually here https://github.com/NixOS/nixpkgs/pull/185719/files#diff-5907dc7a16a18d3c7ce0d98e89c9c340d2fd2686e20a1592371bb1bb11b3ba89R1036